webClient/src/app/app.component.ts

```diff
@@ -395,6 +397,43 @@ export class AppComponent implements AfterViewInit {
   loadZssSettings(): Observable<ZssConfig> {
     return this.http.get(ZoweZLUX.uriBroker.serverRootUri("server/proxies")).map((res: Response) => res.json());
   }
+
+
+  saveSettings() {
+    this.http.put(ZoweZLUX.uriBroker.pluginConfigForScopeUri(this.pluginDefinition.getBasePlugin(), 'user', 'sessions', 'favVT.json'),
```

Comment:
This function should save to the same file we read from. We called it `_defaultVT.json`, and that's a strange name but its still true: it is YOUR default, if you have one, otherwise it is the SERVER's default.
So, no need for `favVT.json`, just use `_defaultVT.json`.